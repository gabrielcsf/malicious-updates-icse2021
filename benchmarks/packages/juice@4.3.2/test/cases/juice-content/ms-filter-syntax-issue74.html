<style>
  .content {
    filter: progidDXImageTransform.Microsoft.gradient(startColorstr='argb(rgba(0,0,0,0.5))', endColorstr='argb(rgba(0,0,0,0))', GradientType=1);
  }
</style>
<div class="content">Hello!</div>