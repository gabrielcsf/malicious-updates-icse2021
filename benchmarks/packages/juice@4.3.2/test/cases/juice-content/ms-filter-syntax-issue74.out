<div class="content" style="filter: progidDXImageTransform.Microsoft.gradient(startColorstr='argb(rgba(0,0,0,0.5))', endColorstr='argb(rgba(0,0,0,0))', GradientType=1);">Hello!</div>